test/acceptance/features/volumeMount.feature

```diff
@@ -0,0 +1,367 @@
+
+    Scenario: Binding is injected as files at the location of SERVICE_BINDING_ROOT env var
+        Given OLM Operator "backend" is running
+        * Generic test application "generic-app-a-d-u-1" is running with binding root as "/var/data"
+        * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: backend-demo
+                annotations:
+                    "service.binding/host": "path={.spec.host}"
+                    "service.binding/port": "path={.spec.port}"
+            spec:
+                host: example.common
+                port: 8080
+            """
+        When Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-backend-vm-01
+            spec:
+                bindAsFiles: true
+                services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: backend-demo
+                    id: bk
+
+                customEnvVar:
+                  - name: MYHOST
+                    value: '{{ .bk.spec.host }}'
+
+                application:
+                    name: generic-app-a-d-u-1
+                    group: apps
+                    version: v1
+                    resource: deployments
+            """
+        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-backend-vm-01" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-backend-vm-01" should be changed to "True"
+        And The env var "BACKEND_HOST" is not available to the application
+        And The env var "BACKEND_PORT" is not available to the application
+        And The env var "MYHOST" is not available to the application
+        And The application env var "SERVICE_BINDING_ROOT" has value "/var/data"
+        And Content of file "/var/data/binding-backend-vm-01/BACKEND_HOST" in application pod is
+            """
+            example.common
+            """
+        And Content of file "/var/data/binding-backend-vm-01/BACKEND_PORT" in application pod is
+            """
+            8080
+            """
+        And Content of file "/var/data/binding-backend-vm-01/MYHOST" in application pod is
+            """
+            example.common
+            """
+
+    Scenario: Binding is injected as file into application at default location
+        Given OLM Operator "backend" is running
+        * Generic test application "generic-app-a-d-u-2" is running
+        * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: backend-demo
```

Comment:
backend-demo is already in use in the previous scenario, please change to something unique.